### DISCUSSION TITLE
Question 2

### DISCUSSION THREAD

**Discussion by Maxi2022gt** [XeroOl/notitg-mirin#81] [General]
how do i move the window? or that funcion isnt in mirin?

**Comment by XeroOl** [comment]
In the future, please use the Discussions tab for asking questions, or ask on discord in the #coding-workshop channel. (https://uksrt.heysora.net/discord or https://discord.gg/wHTFcHdkyg)

**Comment by XeroOl** [comment]
You can set the window position with `DISPLAY:SetWindowPosition(x, y)`. There are a couple of other display functions related to window positions, listed here: https://craftedcart.gitlab.io/notitg_docs/lua_api/rage_display.html#RageDisplay.SetWindowPosition
This is built into NotITG, and not controlled by the Mirin Template.

**Reply by Maxi2022gt** [reply]
Ok! thanks!

**Reply by Maxi2022gt** [reply]
its like https://www.youtube.com/watch?v=qoui8NPe8cc ?

**Reply by XeroOl** [reply]
Yeah, it's the exact same function!

**Reply by Maxi2022gt** [reply]
it worked thank you :D
